lib/redirect.js

```diff
@@ -145,7 +147,7 @@ Redirect.prototype.onResponse = function (response) {
 
   request.emit('redirect')
 
-  request.init()
+  request.init(request.options)
```

Comment:
@falms it might be better, but given the existing code base it's better to keep things consistent instead of adding even more noise.